src/extension/background-script/actions/liquid/enable.ts

```diff
@@ -0,0 +1,78 @@
+import utils from "~/common/lib/utils";
```

Comment:
there is a lot of duplication in all the enable methods now

Reply:
ok! but that was already decided to do. because of the additional checks we need in the prompt. they look same but still they have different actions now because of onboarding